src/components/realtimeAPIUtils.tsx

```
@@ -0,0 +1,343 @@
+            appendValues
+          )
+        } else {
+          console.error('Received invalid audio buffer')
+        }
+      } else {
+        console.error('Received invalid audio buffer')
+      }
+      break
+    case 'response.content_part.done':
+      if (jsonData.part && jsonData.part.transcript) {
+        await processMessage({
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)